src/Components/test/testassets/BasicTestApp/KeyCasesTreeNode.razor

```diff
@@ -0,0 +1,53 @@
+<div class="node">
```

Comment:
Should we have some E2E test that involves inputs to set expectations?
* For example, when we move an input the focus doesn't get lost.
* We could probably check that the cursor/selection stays in place.
* Could we have a test for inputs moving elsewhere in the tree? (like the checkboxes issue that was reported the other day).